### DISCUSSION TITLE
The NVC command?

### DISCUSSION THREAD

**Discussion by omgitsheaven** [kitsuned/PotatoNV#81] [Q&A]
So for starters, I have a Nova 3i, with a Kirin 710. I got my bootloader unlocked by Ministry of Solutions through SigmaKey. So now that my bootloader is unlocked, is it possible that I could retrieve the bootloader unlock code from the phone using the nve command? 

Okay, so using a Root Explorer, I pulled nvme file from /dev/block/platform/hi.mci.0/by-name.

When I search for WVLOCK, which is apparently meant to be the string for the bootloader code. All I get is this: ÝUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUDDDDDDDDDDDDDDDD.

Any clues on what's happening?

**Comment by mashed-potatoes** [comment]
Starting from EMUI 5, the code is not stored in NVME by default.
By the way, I don't recommend using MoS services.

**Reply by omgitsheaven** [reply]
Why though, why not MoS?

By any chance do we know where the code is stored?
